app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -56,6 +56,14 @@ fun HomeScreen(
         }
     }
 
+    // TODO remove in integration ticket
+    val hasConnection by viewModel.hasConnection.collectAsState()
```

Comment:
Can we have the same naming for connectivity detection? `isConnected` or `isNetworkConnected`